genie-web/src/main/java/com/netflix/genie/web/configs/aws/GenieAwsApiAutoConfiguration.java

```diff
@@ -58,11 +58,6 @@
     @Bean
     @ConditionalOnMissingBean(GenieHostInfo.class)
     public GenieHostInfo genieHostInfo() throws UnknownHostException {
-        final String ec2LocalHostName = EC2MetadataUtils.getLocalHostName();
-        if (StringUtils.isNotBlank(ec2LocalHostName)) {
-            return new GenieHostInfo(ec2LocalHostName);
-        }
-
         final String ec2Ipv4Address = EC2MetadataUtils.getPrivateIpAddress();
```

Comment:
Does this work on non-EC2 hosts? i.e. jenkins?
Could we add a unit-test that checks for this string to be an IPv4 string?

Reply:
This auto config is protected by `@ConditionalOnAwsCloudEnvironment` so it won't even be activated if not on an EC2 node there is a different `GenieHostInfo` bean created by default in `GenieApiAutoConfiguration` if this AWS version wasn't used. The fallback case should really never be hit but is there for completeness.

 I could add said unit test but aren't I then really checking the functionality of `EC2MetadataUtils`?